xmtp/libxmtp
# Update Android example app to demonstrate latest features

Updated the Android example app to demonstrate the new signing flow as well as streaming support

To test streaming:
- `cd bindings_ffi && ./gen_kotlin.sh && ./setup_android_example.sh` (no need to rebuild, the binary is in this PR)
- Open Android Studio, and run the app on an emulator
- `cd .. && ./dev/up`
- Re-run the app (currently, Android emulator fights with the dev server for port 5555, so the emulator must be started first - will fix in future)
- Filter logcat logs to 'App'. Observe the wallet address logged, and that streaming was started
- `cd examples/cli` and register a CLI account, create a group, and add the wallet address logged from the example app
- Observe that the logcat logs update in the example app with a description of the new group
